Dockerfile.rootless

```diff
@@ -51,6 +54,7 @@ RUN chown git:git /var/lib/gitea /etc/gitea
 
 COPY docker/rootless /
 COPY --from=build-env --chown=root:root /go/src/code.gitea.io/gitea/gitea /usr/local/bin/gitea
+COPY --from=build-env /go/src/code.gitea.io/gitea/environment-to-ini /usr/local/bin/environment-to-ini
```

Comment:
```suggestion
COPY --from=build-env --chown=root:root /go/src/code.gitea.io/gitea/environment-to-ini /usr/local/bin/environment-to-ini
```

Reply:
Isn't this a rootless docker?

Reply:
yes, but root still exists in docker rootless container (it's just mapped the user id who started the container by default). Since the gitea binary is owned by root it probably makes sense for this to be as well.

Reply:
I see the line above it also runs the same chown.